src/formatter-settings/assets/style.scss

```diff
@@ -34,6 +34,12 @@ $enable-rounded: false;
   }
 }
 
+@media (max-width: 992px) and (min-width: 783px) {
```

Comment:
Here to fix the bug when a setting section has an odd number of settings, in the layout where a line contains two settings, the last setting would be larger than others.